build.gradle.kts

```diff
@@ -11,7 +11,6 @@ buildscript {
     apply(from = "buildSrc/plugins.gradle.kts")
     repositories {
         google()
-        mavenCentral()
```

Comment:
This is required for [Dokka](https://github.com/Kotlin/dokka#using-the-gradle-plugin).

Reply:
Dokka is avaliable in `gradlePluginPortal()`, I aslo execute `./gradlew clean dokkaHtmlMultiModule` for test

![image](https://user-images.githubusercontent.com/10363352/124087785-f243cd00-da84-11eb-9b01-af429b89e3b4.png)

Reply:
This is because `gradlePluginPortal` [implicitly mirrors Jcenter](https://github.com/gradle/gradle/issues/15406) which mirrors `mavenCentral`. This is an implementation detail that's likely to change soon which will make `gradlePluginPortal` stop resolving `mavenCentral` libraries.

Reply:
I get it. Reverted.